test/mocks/get_event_200.json

```diff
@@ -140,6 +174,7 @@
       "data": {
         "result": false,
         "originTimezone": "Europe/Berlin",
+        "originCountry": "unknown",
```

Comment:
Does it make sense that it's unknown when above we know it's Germany?

Reply:
It's a fake data anyway and it is not consistent through the whole example.